mmv1/templates/terraform/encoders/spanner_instance.go.tmpl

```diff
@@ -12,5 +31,11 @@ if obj["name"] == nil {
 } else {
 	newObj["instanceId"] = obj["name"]
 }
+if obj["instanceType"] != nil && obj["instanceType"] == "FREE_INSTANCE" {
+	// when provisioning a FREE_INSTANCE, the following fields cannot be specified
```

Comment:
We should probably throw an error here rather than ignoring the user's value for these fields. If they set them it will cause a permadiff